apple/internal/partials/debug_symbols.bzl

```diff
@@ -307,6 +313,7 @@ def _debug_symbols_partial_impl(
                 dsym_info_plist_template = dsym_info_plist_template,
                 dsym_output_filename = dsym_output_filename,
                 platform_prerequisites = platform_prerequisites,
+                label_name = label_name,
```

Comment:
Can you move this to be alphabetical?

Reply:
oh thanks for noticing that, I missed it!